docs/src/main/sphinx/object-storage/file-system-local.md

```diff
@@ -0,0 +1,45 @@
+# Local file system support
+
+Trino includes support to access a local file system with a catalog using the
+Delta Lake, Hive, Hudi, or Iceberg connectors. The local file system must be a
+local mount point for a shared file system available on all cluster nodes.
+
+Support for local file system is not enabled by default, but can be activated by
+setting the `fs.local.enabled` property to `true` in your catalog configuration
+file.
+
+## General configuration
+
+Use the following properties to configure general aspects of local file system
+support:
+
+:::{list-table}
+:widths: 40, 60
+:header-rows: 1
+
+* - Property
+  - Description
+* - `fs.native-local.enabled`
+  - Activate the support for local file system access. Defaults to `false`. Set
+    to `true` to use local file system and enable all other properties.
+* - `local.location`
```

Comment:
if I dont read things in the code wrong .. the prefix is local or file .. and the property is location.. but there is no config class so I am entirely unsure